modules/t/xref_parser.t

```diff
@@ -180,7 +184,7 @@ my @recognised_sources = (
  "MEROPS; C26.956; -.",
 );
 for my $l (@recognised_sources) {
-  (my $uniprot_elegans_record_extra_line = $uniprot_elegans_record) =~ s/DR(.*?)\n/DR$1\nDR  $l/;
+  (my $uniprot_elegans_record_extra_line = $uniprot_elegans_record) =~ s/DR(.*?)\n/DR$1\nDR   $l/;
```

Comment:
I don't understand this change. Was the regex wrong in the first place, or did the UniProt format change?

Perhaps \s+ would be better than adjusting the amount of spaces.

Reply:
The regex was wrong. Quoting UniProtKB User Manual (https://web.expasy.org/docs/userman.html#entrystruc, at the very end of the section): "The two-character line-type code that begins each line is always followed by three blanks, so that the actual information begins with the sixth character."